pkg/datapath/linux/config/config_test.go

```diff
@@ -297,38 +304,49 @@ func createVlanLink(vlanId int, mainLink *netlink.Dummy, c *C) *netlink.Vlan {
 }
 
 func (s *ConfigSuite) TestVLANBypassConfig(c *C) {
-	oldDevices := option.Config.GetDevices()
-	defer func() {
-		option.Config.SetDevices(oldDevices)
-	}()
+	var devs []*tables.Device
 
 	main1 := createMainLink("dummy0", c)
+	devs = append(devs, &tables.Device{Name: main1.Name, Index: main1.Index})
 	defer func() {
 		netlink.LinkDel(main1)
 	}()
 
+	// Define set of vlans which we want to allow.
+	allow := map[int]struct{}{
```

Comment:
Nit: `map[int]bool` might be easier to work with?

Reply:
done :)